addons/auth_passkey_portal/static/src/css/passkeys_portal.css

```diff
@@ -0,0 +1,53 @@
+.o_passkey {
+    width: calc(50% - 10px);
+    height: 80px;
+    margin-bottom: 20px;
+    border: 1px solid #d8dadd;
+    padding: 8px;
+    position: relative;
+}
+
+.o_passkey small {
+    display: block;
+}
+
+.o_passkey:nth-child(odd) {
+    margin-right: auto;
+}
+
+.o_passkey_name {
+    display: inline-block;
+    font-weight: bold;
+    max-width: calc(100% - 24px);
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+}
+
+.o_passkey_dropdown_btn {
+    position: absolute;
+    top: 4px;
+    right: 4px;
+    background: none;
+    border: none;
+}
+
+.o_passkey_dropdown {
+    position: absolute;
+    right: 0px;
+    top: 28px;
+    background-color: white;
+    border: 1px #d8dadd solid;
+    border-radius: 0.25rem;
+    box-shadow: 0 0.5rem 1rem rgba(0, 0, 0, 0.15);
+    width: 100px;
+    height: 66px;
+    padding-left: 20px;
+    padding-top: 8px;
+}
+
+.o_passkey_dropdown a {
+    color: black;
+    display: block;
+    padding-bottom: 4px;
+}

```

Comment:
I would avoid custom css. Can't you use standard classes ?

For instance it doesn't look like Developer API Keys in the portal interface use any custom CSS:

https://github.com/odoo/odoo/blob/2712e75a9f85847359b26eca7bcc5850e1aab2d2/addons/portal/views/portal_templates.xml#L529-L565
<img width="690" height="255" alt="image" src="https://github.com/user-attachments/assets/06face9f-d5ef-43aa-b0ef-b1a2e9c3745c" />

And it looks better than the style of passkeys